packages/arb-token-bridge-ui/src/hooks/useChainLayers.ts

```diff
@@ -15,11 +15,17 @@ function getChainLayerByChainId(chainId: number): ChainLayer {
   return 'Orbit'
 }
 
-export const useChainLayers = () => {
+export const useChainLayers = ({
+  parentChainIdOverride,
```

Comment:
The purpose of hooks is to get reactive/dynamic data in components. 
If we need to pass overrides of chain-ids that depend per transaction, then it's better that we don't include the override in the hook, and rather, in the UI, simply get the chain-layer details by calling `getChainLayerByChainId(tx.parentChainId/childChainId)` function directly. 
It will keep the usage of this hook pure as well.

Reply:
That's true, although I'll be removing this because we'll use network names instead